contributors/onboarding/staff-onboarding-guide/README.md

```diff
@@ -1,55 +1,51 @@
----
-description: 10% complete.
----
-
 # Staff Onboarding
 
 ## Contents
 
-* [WIP: Staff Onboarding Guide](http://handbook.mattermost.com/people/general-onboarding.html#wip-staff-onboarding-guide)
-  * [Welcome to Mattermost](http://handbook.mattermost.com/people/general-onboarding.html#welcome-to-mattermost)
-  * [How to use our Handbook](http://handbook.mattermost.com/people/general-onboarding.html#how-to-use-our-handbook)
-  * [Our Mission](http://handbook.mattermost.com/people/general-onboarding.html#our-mission)
-  * [Leadership Principles](http://handbook.mattermost.com/people/general-onboarding.html#leadership-principles)
-  * [Products](http://handbook.mattermost.com/people/general-onboarding.html#products)
-  * [Company](http://handbook.mattermost.com/people/general-onboarding.html#company)
-  * [General Onboarding Checklist](http://handbook.mattermost.com/people/general-onboarding.html#general-onboarding-checklist)
-  * [Departmental Onboarding Checklists](http://handbook.mattermost.com/people/general-onboarding.html#departmental-onboarding-checklists)
+* [WIP: Staff Onboarding Guide](https://handbook.mattermost.com/people/general-onboarding.html#wip-staff-onboarding-guide)
+  * [Welcome to Mattermost](https://handbook.mattermost.com/people/general-onboarding.html#welcome-to-mattermost)
+  * [How to use our Handbook](https://handbook.mattermost.com/)
+  * [Our Mission](https://handbook.mattermost.com/company/about-mattermost#mission)
+  * [Leadership Principles](https://handbook.mattermost.com/company/about-mattermost#leadership-principles)
+  * [Products](https://docs.mattermost.com/overview/product.html)
+  * [Company](https://handbook.mattermost.com/company/about-mattermost/business-model)
+  * [General Onboarding Checklist](https://handbook.mattermost.com/contributors/onboarding)
+  * [Departmental Onboarding Checklists](https://handbook.mattermost.com/contributors/onboarding)
```

Comment:
Seems that this should link to "Departmental Onboarding Checklists and Information" section